spec/features/audit/allocation/allocation_spec.rb

```diff
@@ -14,7 +14,7 @@
     create(:allocation, content_item: content_item, user: current_user)
     create(:allocation, content_item: another_content_item, user: another_user)
 
-    visit audits_path
+    visit audits_allocations_path
 
     expect(page).to have_selector(".nav")
```

Comment:
If this test was wrong before we should probably then check that on the page. So maybe check that the page renders the `Assign to...` label for the dropdown, which doesn't appear for any of the other tabs.

Reply:
Addressed in ad69271